.github/workflows/add-lang.yml

```diff
@@ -9,12 +9,29 @@ jobs:
     runs-on: ubuntu-latest
     if: github.event.label.name == 'new language'
     steps:
+
+# Check whether the language has been written correctly:
+      - name: Check issue title format
+        env:
+          ISSUE_TITLE: ${{ github.event.issue.title }}
+          ISSUE_NUM: ${{ github.event.issue.number }}
+        run: |
+          NEW_LANG=$(sed 's/.*\[\([^]]*\)].*/\1/' <<< "$ISSUE_TITLE")
+          if [[ $NEW_LANG =~ ^[a-z]{2}(_[A-Z]{2})?$ ]]; then
+            echo "new_lang=$NEW_LANG" >> $GITHUB_ENV
+            echo "issue_num=$ISSUE_NUM" >> $GITHUB_ENV
+          else
+            echo Language code in wrong format
+            exit 1
```

Comment:
Good idea to have it here. I think we should have a look at all these scripts from a security PoV. I think @hoffie knows some more about bash & security?